test/Microsoft.IdentityModel.TestUtils/TokenValidationExtensibility/Tests/ExtensibilityTheoryData.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Security.Claims;
+using System;
+using Microsoft.IdentityModel.Tokens;
+
+#nullable enable
+namespace Microsoft.IdentityModel.TestUtils.TokenValidationExtensibility.Tests
+{
+    public class ExtensibilityTheoryData : TheoryDataBase
+    {
+        string _tokenHandlerType;
+        SecurityTokenDescriptor? _securityTokenDescriptor;
+
+        internal ExtensibilityTheoryData(
+            string testId,
+            string tokenHandlerType,
+            int extraStackFrames) : base(testId)
+        {
+            ExtraStackFrames = extraStackFrames;
+            TokenHandler = SecurityTokenHandlerForType(tokenHandlerType);
+            _tokenHandlerType = tokenHandlerType;
+
+            ValidationParameters = ValidationParametersSkippingValidations();
+        }
+
+        private IExtensibilityTestingTokenHandler SecurityTokenHandlerForType(string tokenHandlerType)
+        {
+            return tokenHandlerType switch
+            {
+                "JWT" => new JsonWebTokenHandlerWithResult(),
+                "SAML" => new SamlSecurityTokenHandlerWithResult(),
+                "SAML2" => new Saml2SecurityTokenHandlerWithResult(),
+                _ => throw new NotImplementedException(tokenHandlerType)
+            };
+        }
+
+        private SecurityTokenDescriptor PopulateSubjectForSecurityTokenDescriptor(
+            SecurityTokenDescriptor securityTokenDescriptor,
+            string tokenHandlerType)
+        {
+            ClaimsIdentity subject = tokenHandlerType switch
+            {
+                "JWT" => Default.ClaimsIdentity,
+                "SAML" => Default.SamlClaimsIdentity,
```

Comment:
```suggestion
                "SAML" or "SAML2" => Default.SamlClaimsIdentity,
```

Reply:
Thanks, I tried "SAML", "SAML2" and it did not work 😄